keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2547,7 +2547,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "description": "JSON web key operations. For more information, see JsonWebKeyOperation.",
+            "description": "JSON web key operations. The default is all of the operation values; encrypt, decrypt, sign, verify, wrapKey and unwrapKey. For more information, see JsonWebKeyOperation.",
```

Comment:
For default values in models, the default property of the [swagger schemaObject](http://swagger.io/specification/#schemaObject) should be used to annotate the value if it is contractual (i.e. it will not change on the service unless there is a corresponding API version change). Specifying it there will allow tools determine the best way to present the information to the user, which is not possible (or at least very difficult) if you put the values in the English text description.